modules/debts_api/spec/fixtures/pre_submission_fsr/sw_long/minimal_asset_post_transform.json

```diff
@@ -0,0 +1,292 @@
+      "monthlyGrossSalary": "0.00",
+      "deductions": {
+        "taxes": "0.00",
+        "retirement": "0.00",
+        "socialSecurity": "0.00",
+        "otherDeductions": {
+          "name": "",
+          "amount": "0.00"
+        }
+      },
+      "totalDeductions": "0.00",
+      "netTakeHomePay": "0.00",
+      "otherIncome": {
+        "name": "Disability Compensation, Education, Social Security",
+        "amount": "7951.00"
+      },
+      "totalMonthlyNetIncome": "7951.00"
+    },
+    {
+      "veteranOrSpouse": "SPOUSE",
+      "monthlyGrossSalary": "0.00",
+      "deductions": {
+        "taxes": "0.00",
+        "retirement": "0.00",
+        "socialSecurity": "0.00",
+        "otherDeductions": {
+          "name": "",
+          "amount": "0.00"
+        }
+      },
+      "totalDeductions": "0.00",
+      "netTakeHomePay": "0.00",
+      "otherIncome": {
+        "name": "",
+        "amount": "0.00"
+      },
+      "totalMonthlyNetIncome": "0.00"
+    }
+  ],
+  "expenses": {
+    "rentOrMortgage": "0.00",
+    "food": "0.00",
+    "utilities": "0.00",
+    "otherLivingExpenses": {
+      "name": "Clothing",
+      "amount": "6759.00"
+    },
+    "expensesInstallmentContractsAndOtherDebts": "0.00",
+    "totalMonthlyExpenses": "6759.00"
+  },
+  "discretionaryIncome": {
+    "netMonthlyIncomeLessExpenses": "1192.00",
+    "amountCanBePaidTowardDebt": "0.00"
+  },
+  "assets": {
+    "cashInBank": "329.12",
+    "cashOnHand": "125.00",
+    "usSavingsBonds": "135.00",
+    "stocksAndOtherBonds": "3500.00",
+    "realEstateOwned": "0.00",
+    "totalAssets": "4089.12"
+  },
+  "installmentContractsAndOtherDebts": [],
+  "totalOfInstallmentContractsAndOtherDebts": {
+    "originalAmount": "0.00",
+    "unpaidBalance": "0.00",
+    "amountDueMonthly": "0.00",
+    "amountPastDue": "0.00"
+  },
+  "additionalData": {
+    "bankruptcy": {},
+    "additionalComments": "\nIndividual expense amount: Clothing ($6,759.00)"
+  },
+  "applicantCertifications": {
+    "veteranSignature": "Mark  Webb",
+    "veteranDateSigned": "06/10/2024"
+  },
+  "selectedDebtsAndCopays": [
+    {
+      "id": "f4385298-08a6-42f8-a86f-50e97033fb85",
+      "pSSeqNum": "506.00",
+      "pSTotSeqNum": "588.00",
+      "pSFacilityNum": "534",
+      "pSFacPhoneNum": null,
+      "pSTotStatement": "27.00",
+      "pSStatementVal": "0000018255H",
+      "pSStatementDate": "11152019",
+      "pSStatementDateOutput": "11/15/2019",
+      "pSProcessDate": "11112019",
+      "pSProcessDateOutput": "11/11/2019",
+      "pHPatientLstNme": "JONES",
+      "pHPatientFstNme": "TRAVIS",
+      "pHPatientMidNme": "D",
+      "pHAddress1": "1885 SHORE DR S APT 327",
+      "pHAddress2": null,
+      "pHAddress3": null,
+      "pHCity": "SAVANNAH",
+      "pHState": "FL",
+      "pHZipCde": "314104413",
+      "pHZipCdeOutput": "31410-4413",
+      "pHCtryNme": null,
+      "pHAmtDue": "15.00",
+      "pHAmtDueOutput": "15.00&nbsp;&nbsp;",
```

Comment:
**Question (non-blocking):** There are a lot of `&nbsp;` strings in many of the value fields, and I'm worried that when they're processed, they'll mess up the data. Should we remove them?

Reply:
Based on existing submissions these non-breaking spaces seem to predate our changes and are intentional 👍